src/envoy/extensions/wasm/wasm.cc

```diff
@@ -14,13 +14,32 @@
  */
 #include "extensions/common/wasm/wasm.h"
 
+#include "common/stats/utility.h"
 #include "src/envoy/extensions/wasm/context.h"
 
 namespace Envoy {
 namespace Extensions {
 namespace Common {
 namespace Wasm {
 namespace Istio {
+namespace {
+
+// This is unfortunately very complicated because
```

Comment:
Nit: potentially incomplete sentence

Reply:
Keep it as-is :) It's very complicated because... (dramatic silence).

Reply:
lol.  The NB below covers the issue.

Reply:
Removed the comment.